<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CookingDialog</class>
 <widget class="QDialog" name="CookingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="ingredientsListView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>581</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>550</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Main window</string>
   </property>
  </widget>
  <widget class="QLabel" name="ingredientsLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingredients needed</string>
   </property>
  </widget>
  <widget class="QLabel" name="allRecipesLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>All available recipes</string>
   </property>
  </widget>
  <widget class="QLabel" name="pickedRecipesLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>currently picked recipes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>63</y>
     <width>581</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="allRecipesListView"/>
    </item>
    <item>
     <widget class="QListView" name="pickedListView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
